spec/views/users/two_factor_authentication_setup/index.html.erb_spec.rb

```diff
@@ -2,11 +2,14 @@
 
 RSpec.describe 'users/two_factor_authentication_setup/index.html.erb' do
   include Devise::Test::ControllerHelpers
+  include IdvHelper
 
   let(:user) { build(:user) }
   let(:user_agent) { '' }
   let(:show_skip_additional_mfa_link) { true }
+  let(:phishing_resistant_required) { false }
   let(:after_mfa_setup_path) { account_path }
+  let(:return_to_sp_cancel_path) { '/redirect/return_to_sp/cancel' }
```

Comment:
@aduth I'm not so sure about assigning a plain string here. I tried assigning { return_to_sp_cancel_path } but it errored hard. 
Is this correct?

Reply:
@kevinsmaster5 In some contexts, you'll need to explicitly include URL helpers to be able to call those magic URL methods.

Can you try adding this with the other helpers above?

```rb
include Rails.application.routes.url_helpers
```